src/thunderbird_accounts/subscription/tasks.py

```diff
@@ -199,9 +201,56 @@ def paddle_subscription_event(self, event_data: dict, occurred_at: datetime.date
             'current_billing_period_starts_at': current_billing_period_starts_at,
             'current_billing_period_ends_at': current_billing_period_ends_at,
             'user_id': user_uuid,
+            'webhook_updated_at': occurred_at,
         },
     )
 
+    # If a transaction_id is included, then update/create the association to this subscription
+    if transaction_id:
+        Transaction.objects.update_or_create(
+            paddle_id=transaction_id,
+            defaults={
+                'paddle_subscription_id': paddle_id,
+                'subscription_id': subscription.uuid,
+            },
+        )
```

Comment:
Transactions only require a paddle_id and no other data. This is also the only real place we get transaction_id <-> subscription_id from my testing. (It's apparentally in the transaction webhook, but not provided in real data.)